src/components/ProgramRecordsList/ProgramRecordsList.jsx

```diff
@@ -113,7 +113,7 @@ const ProgramRecordsList = () => {
             <div className="d-flex align-items-center pt-3 pt-lg-0">
               <Hyperlink
                 variant="muted"
-                destination={`${getConfig().CREDENTIALS_BASE_URL}/records/programs/${record.uuid}/`}
+                destination={getConfig().SITE_NAME === 'localhost' ? `/${record.uuid}` : `${getConfig().CREDENTIALS_BASE_URL}/records/programs/${record.uuid}/`}
```

Comment:
Related to an earlier comment. We will want to decide on this destination based on a feature flag and not be trying to determine if we're running this on a development environment.

How would we control this feature/work if we _didn't_ want to release this to Production until after we merged it? I don't think we would be able to.

Reply:
Another thing to think about: What if there is a problem that we wanted to fix forward? With a feature flag we can disable the use of the new components and go back to our old behavior (routing to the Credentials IDA) while we fix our issues _without_ needing to revert and remove everything contained in this PR.